tests/Unit/Client/UpdaterTest.php

```diff
@@ -38,22 +39,32 @@ public function testCheckRollbackAttackNoAttack(): void
         // We test lack of an exception in the positive test case.
         $this->expectNotToPerformAssertions();
 
-        // The incoming version is newer than the local version, so no
-        // rollback attack is present.
         $localMetadata = $this->getMockBuilder(MetadataBase::class)->disableOriginalConstructor()->getMock();
         $localMetadata->expects(self::any())->method('getType')->willReturn('any');
         $localMetadata->expects(self::any())->method('getVersion')->willReturn(1);
+
+        $verifier = new class ($localMetadata) extends VerifierBase
```

Comment:
All this should not live in UpdaterTest any more. We could just rename this class as it only tests methods that now are in verifier classes.